en/sql/function/regex_fn.rst

```diff
@@ -381,7 +396,8 @@ Character classes
 
 Character classes syntax matches one of characters or a category of characters within square brackets.
 
-**Individual characters** 
+Individual characters
+~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
I don't know the cause, but the number of "~" must be reduced to create a hyperlink.
```suggestion
~~~~~~~~~~~~~~~~~~~~~
```